<?xml version="1.0" ?>
<launch>
	
	<node name="multi_robot_collision" pkg="stl_aeplanner" type="multi_robot_collision" output="screen">
		<param name="distance_threshold" value="0.2" />
		<param name="visualization" value="true" />
		<param name="robot_name" value="/iris_1" />
	</node>
</launch>
